plugins/wordcompletion/org.mate.pluma.plugins.wordcompletion.gschema.xml.in

```diff
@@ -0,0 +1,15 @@
+<schemalist>
+  <schema gettext-domain="gedit-plugins" id="org.mate.pluma.plugins.wordcompletion" path="/org/mate/pluma/plugins/wordcompletion/">
```

Comment:
It should be `<schema gettext-domain="@GETTEXT_PACKAGE@" id="org.mate.pluma.plugins.wordcompletion" path="/org/mate/pluma/plugins/wordcompletion/">`, then you should rename the file `plugins/wordcompletion/org.mate.pluma.plugins.wordcompletion.gschema.xml.in` to `plugins/wordcompletion/org.mate.pluma.plugins.wordcompletion.gschema.xml.in` and do other changes...